src/librustc/mir/interpret/error.rs

```diff
@@ -302,6 +304,10 @@ impl<O> PanicInfo<O> {
                 "generator resumed after completion",
             GeneratorResumedAfterPanic =>
                 "generator resumed after panicking",
+            AsyncResumedAfterReturn =>
+                "async function resumed after completion",
```

Comment:
```suggestion
                "`async fn` resumed after completion",
```

Reply:
Just confirming that this should be a ` and not a '